packages/native/src/components/LinearGradient.tsx

```diff
@@ -24,14 +24,14 @@ const LinearGradientComponent = ({
   startY = 0,
   endX = 100,
   endY = 100,
-  style,
+  style: propStyle,
 }: LinearGradientComponentProps) => {
-  const { viewStyles } = extractStyles(style);
+  const { viewStyles: style } = extractStyles(propStyle);
   const colors = [color1, color2, color3].filter((color) => color) as string[];
   const start = { x: startX / 100, y: startY / 100 };
   const end = { x: endX / 100, y: endY / 100 };
   return (
-    <LinearGradient colors={colors} start={start} end={end} style={viewStyles}>
+    <LinearGradient {...{ colors, start, end, style }}>
```

Comment:
```suggestion
    <LinearGradient colors={colors} start={start} end={end} style={style}>
```

Reply:
@peter unless there is a perf diff, this is just style now.

Reply:
@peter see latest commit